sagar/data/Lab_1sagar.ipynb

```diff
@@ -0,0 +1,1314 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    plt.plot(x, cos_y, label='cos(x)', color='yellow', linestyle='--')
</pre><p>yellow color doesnot have proper contrast over white background.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>